src/LanguageServer/Terminator.cs

```diff
@@ -117,6 +115,6 @@ async void ParentProcess_Exit(object sender, EventArgs args)
         /// <summary>
         ///     Terminate the current process.
         /// </summary>
-        void Terminate() => Environment.Exit(9);
+        private static void Terminate() => Environment.Exit(9);
```

Comment:
Probably also better to extract magic constant `9`, but I just don't know what it means, so left it "as is"

Reply:
Yes, it's a magic number but you're right - it should be a constant so it's a little less magic 🙂